File: C/Math/1009_complement_of_base_10_integer.c
int bitwiseComplement(int N){
    int powerTwo(int two, int N) {
        // Nearest multiple of 2
        while (two <= N) {
            two *= 2;
        }
        return two;
    }
    
    return powerTwo(2, N) - N - 1;
}

File: Swift/Math/1009_complement_of_base_10_integer.swift
class Solution {
    func bitwiseComplement(_ N: Int) -> Int {
        var compl = 2
        
        // Nearest multiple of 2
        while (compl <= N) {
            compl = compl * 2
        }
        
        return compl - N - 1
    }
}
